shenyu-client/shenyu-client-http/shenyu-client-springcloud/src/main/java/org/apache/shenyu/client/springcloud/init/ContextRegisterListener.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.shenyu.client.springcloud.init;
 
+import com.alibaba.nacos.api.utils.NetUtils;
 import lombok.extern.slf4j.Slf4j;
```

Comment:
I think is a bad idea to use nacos's util here.

Reply:
Yes, you are right, i should  use org.apache.shenyu.common.utils.IpUtils.getHost() instead of using NetUtils.localIP(), right?